be/src/storage/rowset/segment_options.h

```diff
@@ -58,6 +58,7 @@ class SegmentReadOptions {
     int chunk_size = DEFAULT_CHUNK_SIZE;
 
     const ColumnIdToGlobalDictMap* global_dictmaps = &EMPTY_GLOBAL_DICTMAPS;
+    int function_version = 3;
```

Comment:
You should give comments to explain what do this field mean
And what the number value stand for.

Reply:
fixed